Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -967,7 +967,7 @@
   #define NUM_SERIAL 0
   #undef BAUD_RATE_GCODE
 #endif
-#if SERIAL_PORT == -1 || SERIAL_PORT_2 == -1
+#if SERIAL_PORT == -1 || SERIAL_PORT_2 == -1 || SERIAL_PORT_2 == -1
```

Comment:
Typo here, you probably meant SERIAL_PORT_3

Reply:
yip... its getting to late...  will fix and stop for the night